### PR TITLE
Test module importing on Travis

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

**Comment by goanpeca** [comment]
Only 5 commit to get it right :wink:  hehe

**Comment by blink1073** [comment]
Travis is *so* much fun...

File: continuous_integration/travis_script.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  fi
+  if [[ $f == spyderlib/utils/qthelpers.py ]]; then
+    continue
+  fi
+  if [[ $f == spyderlib/utils/windows.py ]]; then
+    continue
+  fi
+  if [[ $f == spyderlib/widgets/*.py ]]; then
+    continue
+  fi
```
Comment (ccordoba12) [inline continuous_integration/travis_script.sh]: If you are skipping `widgets` and `plugins`, what are exactly testing here?

Sorry, I don't get it :-)

**Comment by ccordoba12** [comment]
@blink1073, could you explain what's the purpose of this PR? It's not that clear to me :-)

**Comment by blink1073** [comment]
A lot of the modules have `assert` statements in them when run as `__main__`, this was a start at at least making sure those were still valid.  I ran in to trouble with QApplication errors with `widgets` and `plugins`, so I skipped those for now.

**Comment by goanpeca** [comment]
@blink1073 are the errors by any chance qtawesome related??

**Comment by blink1073** [comment]
For instance, `widgets/arraybuilder` brings up a dialog that would need to be closed.  

```
+ python spyderlib/plugins/__init__.py
QPixmap: Must construct a QApplication before a QPaintDevice
```

```
+ python spyderlib/qt/QtCore.py
Traceback (most recent call last):
  File "spyderlib/qt/QtCore.py", line 9, in <module>
    if os.environ['QT_API'] == 'pyqt5':
  File "/Users/ssilvester/anaconda/lib/python3.4/os.py", line 633, in __getitem__
    raise KeyError(key) from None
KeyError: 'QT_API'
```

**Comment by ccordoba12** [comment]
Thanks @blink1073 for the clarification. This is really important work because with it we can also start testing widgets and plugins, by using @goanpeca's trick of setting a timer to close a `QApplication` after a certain time! :-)

Thanks a lot for it!!

**Comment by ccordoba12** [comment]
There are errors detected by Travis after this PR, but I'll fix them on the other PR I'm working on :-)

**Pull request merged**